tests/unittests/camera_tests.py

```
@@ -2309,5 +2309,130 @@ def test_extrinsics_type_preserved_after_reload(self):
+            fy=400.0,
+            cx=320.0,
+            cy=240.0,
+            k1=0.2,
+            k2=-0.1,
+        )
+        image = np.random.randint(0, 255, (480, 640, 3), dtype=np.uint8)
+
+        result_alpha0 = intrinsics.undistort_image(image, alpha=0.0)
+        result_alpha1 = intrinsics.undistort_image(image, alpha=1.0)
+
+        self.assertEqual(result_alpha0.shape, image.shape)
```

Ruff (0.14.10)

Use a regular `assert` instead of unittest-style `assertEqual`

Replace `assertEqual(...)` with `assert ...`

(PT009)